csm-1.5/CASMTRIAGE-6796-noos-repo/README.md

```diff
@@ -0,0 +1,29 @@
+# CSM 1.5.0 Tarball Noos Repository
+
+This hotfix is exclusive to the 1.5.0 fresh install.
```

Comment:
Could you please add something like "Apply it after tarball is downloaded, but before it is unpacked"? Maybe even highlight is somehow. We definitely need to highlight the right point in time when hotfix needs to be applied.

Reply:
I've highlighted this in the field notice ([Hot Fix.docx](https://github.com/Cray-HPE/csm-hotfixes/files/14859934/Hot.Fix.docx)), as well as the docs-csm PR: https://github.com/Cray-HPE/docs-csm/pull/4868/files

I'll add more emphasis here too.